sky/utils/cli_utils/status_utils.py

```diff
@@ -188,7 +272,7 @@ def show_cost_report_table(cluster_records: List[_ClusterCostReportRecord],
 
     if cluster_records:
         if reserved_group_name is not None:
-            autostop_minutes = spot.SPOT_CONTROLLER_IDLE_MINUTES_TO_AUTOSTOP
+            autostop_minutes = backend_utils.CONTROLLER_IDLE_MINUTES_TO_AUTOSTOP
```

Comment:
Maybe better to keep the AUTOSTOP constant in `*.constants` file?

Reply:
Now it is in `sky/utils/controller_utils.py`. Does this sound great to you?